CHANGELOG.md

```diff
@@ -14,6 +14,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Fixed bug with request headers in `@cumulus/launchpad-auth` causing Launchpad token requests to fail
 
+- **CUMULUS-1955**
```

Comment:
If there are changes being made to `aws-client/S3`, those should be noted here as well.

Reply:
Will do.